Tutorials/CNTK_101_LogisticRegression.ipynb

```diff
@@ -145,15 +145,20 @@
   {
    "cell_type": "code",
    "execution_count": 5,
```

Comment:
Done.  Please let me know if there are still any problems.  Jupyter notebooks can be tricky here.